src/wp-includes/default-filters.php

```diff
@@ -422,6 +422,7 @@
 add_action( 'do_all_pings', 'generic_ping', 10, 0 );
 add_action( 'do_robots', 'do_robots' );
 add_action( 'do_favicon', 'do_favicon' );
+add_action( 'wp_before_include_template', 'wp_start_template_enhancement_output_buffer', 10, 1 );
```

Comment:
Defaults not required.

```suggestion
add_action( 'wp_before_include_template', 'wp_start_template_enhancement_output_buffer' );
```